### PR TITLE
Ctctraders 962

### PR DESCRIPTION
**Pull request by garethgooch**
Adding in pre-lodge question page, updated Navigation and CYA

### CODE REVIEW

**Review by alexflav23 — APPROVED** [review]

File: app/utils/MovementDetailsCheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -19,11 +19,28 @@ package utils
...
+    answer =>
+      Row(
+        key   = Key(msg"preLodgeDeclaration.checkYourAnswersLabel", classes = Seq("govuk-!-width-one-half")),
+        value = Value(yesOrNo(answer)),
+        actions = List(
+          Action(
+            content            = msg"site.edit",
+            href               = routes.PreLodgeDeclarationController.onPageLoad(lrn, CheckMode).url,
+            visuallyHiddenText = Some(msg"site.edit.hidden".withArgs(msg"preLodgeDeclaration.checkYourAnswersLabel")),
+            attributes         = Map("id" -> s"""change-containers-used""")
```
Comment (chris-ianson) [inline app/utils/MovementDetailsCheckYourAnswersHelper.scala:38]: Do we need an id on this?

File: conf/app.routes [hunk 2]
```diff
@@ -336,6 +336,11 @@ POST       /:lrn/add-items/item-details/:index/item-description
...
 POST       /:lrn/add-items/item-details/:index/change-item-description                   controllers.addItems.ItemDescriptionController.onSubmit(lrn: LocalReferenceNumber, index: Index, mode: Mode = CheckMode)
 
+GET        /:lrn/movement-details/preLodgeDeclaration                        controllers.movementDetails.PreLodgeDeclarationController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = NormalMode)
+POST       /:lrn/movement-details/preLodgeDeclaration                        controllers.movementDetails.PreLodgeDeclarationController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = NormalMode)
+GET        /:lrn/movement-details/changePreLodgeDeclaration                  controllers.movementDetails.PreLodgeDeclarationController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = CheckMode)
+POST       /:lrn/movement-details/changePreLodgeDeclaration                  controllers.movementDetails.PreLodgeDeclarationController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = CheckMode)
```
Comment (chris-ianson) [inline conf/app.routes:342]: Need to update all the uris to the final values

**Review by chris-ianson — CHANGES_REQUESTED** [review]
Couple of comments

File: app/utils/MovementDetailsCheckYourAnswersHelper.scala (hunk 1 above)
Comment (garethgooch) [reply app/utils/MovementDetailsCheckYourAnswersHelper.scala:38]: I would have to double check but we often use the id for journey tests to enable it to be selected

Comment (chris-ianson) [reply app/utils/MovementDetailsCheckYourAnswersHelper.scala:38]: If we're keeping can we change it to something more relevant

**Review by chris-ianson — APPROVED** [review]

**Pull request merged**